webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -664,6 +667,88 @@ const AdminDefinition: AdminDefinitionType = {
                 },
                 restrictedIndicator: getRestrictedIndicator(true, LicenseSkus.Enterprise),
             },
+            access_control_policy_details_edit: {
```

Comment:
We need to add the feature discovery screen for when the license is not applied or lower sku - see Groups example https://github.com/mattermost/mattermost/blob/enterprise-advanced-upsells/webapp/channels/src/components/admin_console/admin_definition.tsx#L518 and https://github.com/mattermost/mattermost/blob/enterprise-advanced-upsells/webapp/channels/src/components/admin_console/feature_discovery/features/groups.tsx#L13

You can probably reuse the same SVG as groups per the design https://www.figma.com/design/UCCKYrwLe2Bs2xYqMAEZCI/Attribute-Based-Channel-Access?node-id=5809-421212&t=UTH7kRke726K5V18-4